controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +613,42 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
```

Comment:
What's the difference between 0 & -1 -> both are actually false, why not using bool?

Reply:
There's a difference between "not configured" and false,
since false means it was configured and set to false.
Currently the notion of having "not configured" in the BML is to signify the client that that parameter is not persistent according to the DB

Reply:
> What's the difference between 0 & -1 -> both are actually false, why not using bool?

@yaniv-yitzhak, actually, '-1' is counts as "true".
'true' is defined as not "0".